tests/test_api.py

```diff
@@ -69,6 +69,60 @@ def test_get_version(self):
         res = self.assertGetResponse("/version", requests.codes.ok, headers=get_auth_header())
         self.assertEquals(res.json['version_info']['version'], os.environ['DSS_VERSION'])
 
+    @testmode.standalone
+    def test_put_file_pattern(self):
+        """
+        Tests the regex pattern filtering on the PUT file/{uuid} endpoint.
+
+        Ensures that paths with leading slashes (absolute paths) and
+        relative path shortcuts (".", "~/", and "..") are disallowed.
+        """
+
+        examples_of_bad_paths = ['.', '..',
```

Comment:
are `\\` or `//` allowed, and should `good..path` be allowed?

Reply:
`//` is allowed on a normal Linux system.  `good..path` is weird but valid on most sytems I believe.  We should all talk about what and should not be allowed on Monday.